src/hls-adapter.js

```diff
@@ -209,9 +209,9 @@ export default class HlsAdapter extends BaseMediaSourceAdapter {
    * @private
    */
   _parseTracks(data: any): Array<Track> {
-    let audioTracks = this._parseAudioTracks(data.audioTracks || []);
-    let videoTracks = this._parseVideoTracks(data.levels || []);
-    let textTracks = this._parseTextTracks(this._hls.subtitleTracks || []);
+    const audioTracks = this._parseAudioTracks(data.audioTracks || []);
```

Comment:
why not take audioTracks from [hls.audioTracks](https://github.com/video-dev/hls.js/blob/master/doc/API.md#hlsaudiotracks) as well? a bit strange we take other data but not this

Reply:
Good point.
Done

Reply:
👍 did you check all data is actually available at that point? should be but better ask now...

Reply:
Better to be safe then sorry :)
Sure I've checked